### PR TITLE
Add random message about /donate

### PR DESCRIPTION
**Pull request by Lucyucy**
Adds a random message about the use of /donate

### CODE REVIEW

File: mods/other/random_messages/init.lua [hunk 1]
```diff
@@ -64,7 +64,8 @@ function random_messages.read_messages()
 		"To check all team members use /team.",
 		"You can capture multiple enemy flags at once!",
 		"Consider joining our Discord server at https://discord.gg/vcZTRPX",
-		"You can press sneak while jumping, to jump up two blocks."
+		"You can press sneak while jumping, to jump up two blocks.",
+		"Use /donate <playername> <amount> to reward a team-mate for their work."
```
Comment (magnetar47) [inline mods/other/random_messages/init.lua:68]: > ... to reward a team-mate

This sounds a little ambiguous, IMO. Should we mention that the reward is score?

Comment (Lucyucy) [reply mods/other/random_messages/init.lua:68]: Could change it to /donate <playername> <score> ?

Comment (Lucyucy) [reply mods/other/random_messages/init.lua:68]: ... /donate playername score I meant. This PR can be closed anyways as my new one includes it already

Comment (magnetar47) [reply mods/other/random_messages/init.lua]: Okay this looks better now.

**Review by magnetar47 — APPROVED** [review]

**Pull request merged**